edgelet/iotedge/src/config/import/mod.rs

```diff
@@ -269,9 +269,7 @@ fn execute_inner(
                         global_endpoint,
                         id_scope: scope_id,
                         attestation: common_config::super_config::DpsAttestationMethod::X509 {
-                            // TODO: Remove this when IS supports registration ID being optional for DPS-X509
-                            registration_id: registration_id
-                                .ok_or_else(|| "registration ID is currently required")?,
+                            registration_id: registration_id,
```

Comment:
```diff
-registration_id: registration_id,
+registration_id,
```

(I thought rustfmt did that automatically.)

Reply:
Apparently not, I ran it and nothing happened. Anyways, I fixed it.